modules/xmpp/ChatRoom.js

```diff
@@ -266,8 +267,10 @@ export default class ChatRoom extends Listenable {
             if (this.password) {
                 pres.c('password').t(this.password).up();
             }
-            if (this.options.billingId) {
-                pres.c('billingid').t(this.options.billingId).up();
+
+            // send the machineId with the initial presence
+            if (this.moderator.targetUrl) {
```

Comment:
Why not send it always, so we can eventually remove the backend code that reads it?

Reply:
So sending a that information that way is out of spec so I wanted it to work only when this mode is enabled.

Reply:
But if you think we will enable this http request by default in the future, let's remove it ... if some needs to disable it in the future can create a flag to disable it. WDYT?